spotbugs/src/main/java/edu/umd/cs/findbugs/util/MutableClasses.java

```diff
@@ -15,13 +15,20 @@
 
     private static final Set<String> KNOWN_IMMUTABLE_CLASSES = new HashSet<>(Arrays.asList(
             "java.lang.String", "java.lang.Integer", "java.lang.Byte", "java.lang.Character",
-            "java.lang.Short", "java.lang.Boolean", "java.lang.Long", "java.lang.Double",
-            "java.lang.Float", "java.lang.StackTraceElement", "java.math.BigInteger",
+            "java.lang.Short", "java.lang.Boolean", "java.lang.Class", "java.lang.Long", "java.lang.Double",
+            "java.lang.Float", "java.lang.Runtime.Version", "java.lang.StackTraceElement", "java.math.BigInteger",
             "java.math.Decimal", "java.io.File", "java.awt.Font", "java.awt.BasicStroke",
             "java.awt.Color", "java.awt.GradientPaint", "java.awt.LinearGradientPaint",
             "java.awt.RadialGradientPaint", "java.Cursor.", "java.util.UUID", "java.net.URL",
             "java.net.URI", "java.net.Inet4Address", "java.net.Inet6Address", "java.net.InetSocketAddress",
-            "java.security.Permission"));
+            "java.math.BigDecimal", "java.math.BigInteger",
+            "java.security.Permission",
```

Comment:
`java.security.Permission` is `abstract` so it's not immutable.

https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/security/Permission.html

Reply:
This PR does not add `java.security.Permission` - it was there before, added in commit 2e09a8daf5c1dc.

Shall I add a commit to this PR removing it?

Reply:
I added a commit removing `java.security.Permission`

Hopefully this PR can now be merged :crossed_fingers: